translations/data.fr-FR

```diff
@@ -94,22 +94,22 @@
       "transparency": "Transparence"
     },
     "locate": {
-      "metersUnit": "",
-      "feetUnit": "",
-      "popup": "",
+      "metersUnit": "mètre",
+      "feetUnit": "pied",
+      "popup": "Dans un préimètre de {distance} {unit} de ce point",
```

Comment:
```suggestion
      "popup": "Dans un périmètre de {distance} {unit} de ce point",
```

Reply:
Yes thanks for correcting the typo :)

Reply:
Thanks for updating the translation :+1:!